package.json

```diff
@@ -63,21 +64,27 @@
     "webpack-dev-server": "^2.9.7"
   },
   "jest": {
-    "testPathIgnorePatterns": [
-      "/node_modules/",
-      "/config/webpack/test.js"
-    ],
-    "setupFiles": [
-      "./app/javascript/support/requestAnimationFrameTestShim.js"
-    ],
+    "testPathIgnorePatterns": ["/node_modules/", "/config/webpack/test.js"],
+    "setupFiles": ["./app/javascript/support/requestAnimationFrameTestShim.js"],
     "setupTestFrameworkScriptFile": "./app/javascript/support/testSetup.js"
   },
   "scripts": {
-    "lint": "npm run lint:styles && npm run lint:js",
-    "lint:js": "eslint ./app ./config ./spec *.js",
-    "lint:styles": "sass-lint -c .sass-lint.yml 'app/assets/stylesheets/**/*.scss' -v -q",
+    "lint": "yarn lint:styles && yarn lint:js && yarn lint:prettier",
```

Comment:
The [`npm-run-all`](https://www.npmjs.com/package/npm-run-all) package can be used here for convenience.